playwright.config.ts

```diff
@@ -19,7 +19,14 @@ export default defineConfig({
   projects: [
     {
       name: 'chromium',
-      use: { ...devices['Desktop Chrome'] }
+      use: { ...devices['Desktop Chrome'] },
+      testMatch: /.*\.spec\.ts/,
+      testIgnore: /api-integration\.spec\.ts/ // Exclude API tests from the chromium project
```

Comment:
> > I think we should add the `pnpm test:api` command to the pipeline
> 
> `playwright test` run all playwright tests (_api-integration.spec.ts_ & _end-to-end.spec.ts_) But do you think it still should be a separate step?

I assumed this step ignored it, but I can see below that it is only for the grouping, so everything is okay. :)